charts/backstage/values.yaml

```diff
@@ -346,7 +346,7 @@ test:
     tag: latest
 
 orchestrator:
-  enabled: false
+  enabled: true
```

Comment:
I guess I misunderstood your point.. We should not enable it by default (not yet at least). But the Snyk workflow can enable it explicitly when it renders the Chart.

Reply:
```suggestion
  enabled: false
```